column.go

```diff
@@ -92,14 +92,18 @@ func (c *Column) Column(name string) *Column {
 
 // Pages returns a reader exposing all pages in this column, across row groups.
 func (c *Column) Pages() Pages {
+	return c.PagesWithReader(c.file.reader)
+}
+
+func (c *Column) PagesWithReader(reader io.ReaderAt) Pages {
```

Comment:
Let's name this PagesFrom for consistency?

Reply:
🤦 my. bad, fixed